TYPO3.Flow/Classes/TYPO3/Flow/Package/PackageFactory.php

```diff
@@ -41,10 +43,15 @@ public function create($packagesBasePath, $packagePath, $packageKey, $composerNa
         }
 
         $packageClassName = Package::class;
+        $packageClassExists = true;
         if (!empty($packageClassInformation)) {
             $packageClassName = $packageClassInformation['className'];
             $packageClassPath = Files::concatenatePaths(array($absolutePackagePath, $packageClassInformation['pathAndFilename']));
-            require_once($packageClassPath);
+            $packageClassExists = @include_once($packageClassPath);
+        }
+
+        if ($packageClassExists === false) {
```

Comment:
The `require_once()` used to include a package class _if it existed_. If not, the generic package class was used. Now an exception is thrown, which seems wrong… Let's see if that fixes the tests if changed.